public/pastMeetup.html

```diff
@@ -138,7 +144,7 @@
         <template x-if="meetup">
           <div
             id="event"
-            class="mx-auto max-w-4xl px-4 text-center text-5xl font-bold capitalize sm:pl-0 lg:w-full lg:text-7xl"
+            class="mx-auto max-w-4xl px-4 text-center text-[34px] font-bold capitalize xs:text-[40px] sm:pl-0 lg:w-full lg:text-7xl"
```

Comment:
```suggestion
class="mx-auto max-w-4xl px-4 text-center text-4xl font-bold capitalize xs:text-5xl sm:pl-0 lg:w-full lg:text-7xl"
```

Reply:
The values You provided are slightly bigger than the ones I set, which causes some problems on smaller mobile devices.
![Screenshot from 2024-07-10 21-23-23](https://github.com/blumilksoftware/lmt/assets/39692087/5570ceac-8e7c-461a-bb97-64b8b1bf39c6)
![Screenshot from 2024-07-10 21-23-40](https://github.com/blumilksoftware/lmt/assets/39692087/b3503bb7-e320-4c22-b96f-0f8ad0c99c5d)

Reply:
I missed changes due to page refreshing, so it's ok right now